### PR TITLE
Create network_ip_availability.md

### PR DESCRIPTION
**Pull request by LeslieLundquist**
added from education session by JLK

### CODE REVIEW

**Comment by LeslieLundquist** [comment]
@j2sol here is another doc from your information session this morning. Can you double-check, please?

File: _neutron/network_ip_availability.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+featured: false
+weight: 3
+tags: [neutron, IP, availability]
+dateAdded: July 7, 2016
+author: Leslie Lundquist
+---
+
+## How Can I Check My Network IP Availability?
+
+Many customers have asked how they can check to see how many Floating IPs are left in their availability pool. There’s now an API call for resource allocation reporting:
```
Comment (omgjlk) [inline _neutron/network_ip_availability.md]: There's now an API call and a client call for...

File: _neutron/network_ip_availability.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+```
+$ network-ip-availabilities
```
Comment (omgjlk) [inline _neutron/network_ip_availability.md]: `neutron net-ip-availability-list`

File: _neutron/network_ip_availability.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+```
+This command gives information about how many floating IPs are used, and how many are still available, of these “consumables.”
+
+The command does not give information about bandwidth usage, that information is provided by Ceilometer.
```
Comment (omgjlk) [inline _neutron/network_ip_availability.md]: I would leave this sentence off all together.

Comment (omgjlk) [inline _neutron/network_ip_availability.md]: And link to http://docs.openstack.org/developer/neutron/devref/network_ip_availability.html for more information, as well as http://docs.openstack.org/cli-reference/neutron.html#neutron-net-ip-availability-list

**Comment by LeslieLundquist** [comment]
@j2sol your comments added. Mucho appreciado!

**Pull request merged**